network/Message.h

```diff
@@ -51,13 +51,16 @@ namespace Moderator {
   * misbehave as well.) */
 class FO_COMMON_API Message {
 public:
-    enum Parts : std::size_t {TYPE = 0, SIZE, Parts_Count};
+    enum Parts : std::size_t {TYPE = 0, SIZE, CPR_USED, Parts_Count};;
 
     typedef std::array<int, Parts::Parts_Count> HeaderBuffer;
 
     constexpr static std::size_t HeaderBufferSize =
         std::tuple_size<HeaderBuffer>::value* sizeof(HeaderBuffer::value_type);
 
+    /** Commpression is only performed on message data of at least this size in bytes. */
+    const static int COMPRESSION_THRESHOLD = 64;
```

Comment:
```suggestion
    static constexpr int COMPRESSION_THRESHOLD = 64;
```

Reply:
Change made in https://github.com/freeorion/freeorion/pull/3854/commits/4ead6795ce2b51bce839955a679750840ada4831.